### DISCUSSION TITLE
Compile errors on Windows

### DISCUSSION THREAD

**Discussion by chylex** [JetBrains/ideavim#389] [General]
Wondering if anyone else ran into this - `KeywordOptionTest.java` has been throwing compile errors because of special characters, but it seems it's only happening on Windows.

```
assertIsKeyword('Ā');
// unclosed character literal
// unclosed character literal
// not a statement
```

Using OpenJDK 16 to build. I've replaced them with `\uXXXX` literals in my fork so that I can compile again, but I'm not sure why it's happening and whether it's something on my end.

**Comment by AlexPl292** [comment]
Hi, what is the value of `javaVersion` in gradle.properties file? Do you compile the project using `gradle build` or using IntelliJ IDEA?

**Reply by chylex** [reply]
I haven't touched `javaVersion`, it's set to 11 as of a14252493f1ccac7cd5bd71930315546239cb91d. I was incorrect about the JDK, the project is actually set to Amazon Corretto 11 that I have for building IntelliJ. Both `gradle build` and IntelliJ fail.

![obrazek](https://user-images.githubusercontent.com/3685160/139922789-8f42e66c-b0b4-4360-ad6b-71cafb4a4cfc.png)

**Reply by chylex** [reply]
Adding
```
org.gradle.jvmargs='-Dfile.encoding=UTF-8'
```
to `gradle.properties` fixed it.
